rts/Lua/LuaSyncedRead.cpp

```diff
@@ -7038,6 +7054,19 @@ int LuaSyncedRead::GetFeatureLastAttackedPiece(lua_State* L)
  *
  * @function Spring.GetFeatureCollisionVolumeData
  * @param featureID integer
+ * @return {
```

Comment:
Make this a `@class`, for now this should be good enough, maybe in the future `@alias` could work well?

See [this](https://github.com/beyond-all-reason/spring/blob/master/rts/Lua/LuaSyncedRead.cpp#L1426-L1436) example.